packages/@lwc/module-resolver/src/utils.ts

```diff
@@ -68,17 +68,17 @@ export function getModuleEntry(
 
 export function normalizeConfig(
     config: Partial<ModuleResolverConfig>,
-    scope: string
+    scope?: string
 ): ModuleResolverConfig {
-    const rootDir = config.rootDir ? path.resolve(config.rootDir) : process.cwd();
+    const rootDir = config.rootDir ? path.resolve(config.rootDir) : scope || process.cwd();
     const modules = config.modules || [];
     const normalizedModules = modules.map((m) => {
         if (!isObject(m)) {
             throw new LwcConfigError(
```

Comment:
The cause of the error is not directly related to the `scope` param. Hence, IMO it should just be informational and not directly affect the error message itself.
```suggestion
           throw new LwcConfigError(
                scope ? 
                `Invalid module record. Module record must be an object, instead got ${JSON.stringify(
                    m
                )}. (Inline config used as no scope provided)` :
                `Invalid module record. Module record must be an object, instead got ${JSON.stringify(
                    m
                )}.`,
                { scope: scope || process.cwd() }
            );
```